apps/financial-aid/api/src/app/modules/municipality/dto/createMunicipality.input.ts

```diff
@@ -1,15 +1,19 @@
 import { Allow } from 'class-validator'
 
 import { Field, InputType } from '@nestjs/graphql'
-import type { CreateMunicipality } from '@island.is/financial-aid/shared/lib'
+import { CreateStaffInput } from '../../staff'
 
 @InputType()
-export class CreateMunicipalityInput implements CreateMunicipality {
+export class CreateMunicipalityInput {
```

Comment:
Since this is not implementing `CreateMunicipality` are we still using that variable anywhere? And if not should we throw it in the trash 🗑️

Reply:
Yess I am using it in backend here for example: 

`
  createMunicipality(
    createMunicipality: CreateMunicipality,
    createAdmin: CreateStaff,
  ): Promise<Municipality> {
    return this.post('municipality', {
      municipalityInput: createMunicipality,
      adminInput: createAdmin,
    })
  }
`